apps/wing/src/commands/test.ts

```diff
@@ -200,7 +200,11 @@ async function checkTerraformStateIsEmpty(synthDir: string) {
       );
     }
   } catch (err) {
```

Comment:
maybe add these types to line 202, that way you can should be able to avoid casting as `any` (will still need to verify the types though, but TS should understand your intent)

```suggestion
  } catch(err: string | { stderr: string }) {
```

Reply:
even better if we know the "real" type of the error with stderr (looks like an `exec` of some kind?)

Reply:
Found the type!! it's just a regular `Error` :) 
@Chriscbr was there a reason it accessed `(err as any).stderr` earlier?

Reply:
It's possible the real error is actually a subclass of `Error`, so it can have that additional property.

Reply:
added a unit test to this method: checkTerraformStateIsEmpty